app/controllers/site_controller.rb

```diff
@@ -106,6 +106,11 @@ def about
     @locale = params[:about_locale] || I18n.locale
   end
 
+  def communities
+    OsmCommunityIndex::LocalChapter.add_to_i18n # this should be called on app init
```

Comment:
One thing tho - I would really appreciate any advice on where to move this to, so that it's called on initialisation of the website.

Reply:
@gravitystorm I'd still really appreciate your input on where this can be moved to